models/reviewsModel.js

```diff
@@ -19,4 +19,22 @@ const fetchReviews = () => {
     });
 };
 
-module.exports = { fetchReviews };
+const fetchReviewsById = (reviewId) => {
+  const changeToNumber = +reviewId;
```

Comment:
You don't have to do this for the review id as it's a number by default as it's serial

Reply:
Thanks for the feedback August. 
However when i console log typeof reviewId on line 23, the first line of the func, i get string. So how would my isNaN be able to differentiate between if someone passed a string or number?